connect/template_projects/tests.py

```diff
@@ -1,52 +1,77 @@
-import json
-from django.test import TestCase, RequestFactory
-from connect.template_projects.models import TemplateType
-from connect.api.v1.tests.utils import create_user_and_token
-from connect.api.v2.template_projects.views import TemplateTypeViewSet
+from django.test import TestCase
+from connect.template_projects.models import TemplateType, TemplateAI, TemplateFeature
+
+
 
 # Create your tests here.
```

Comment:
its interesting that you remove the comment

Reply:
Done